mantle/platform/api/gcloud/compute.go

```diff
@@ -33,8 +34,56 @@ func (a *API) vmname() string {
 	return fmt.Sprintf("%s-%x", a.options.BaseName, b)
 }
 
+// ["local-ssd:interface=NVME,size=375"]
```

Comment:
For consistency, maybe let's have it be more similar to the QEMU one, where the main non-optional item is the size, and then the optional attributes after. So e.g. `["5G:type=TYPE"]`. We can default `type` to `local-ssd` so that just `5G` works.